pmm/kubernetes-cluster-staging.groovy

```diff
@@ -27,6 +27,14 @@ pipeline {
             choices: ['1.23.1', '1.22.1', '1.21.1', '1.20.1'],
             description: 'Select Kubernetes version',
             name: 'KUBE_VERSION')
+        choice(
+            choices: ['none', 'v1.8.0', 'v1.9.0', 'v1.10.0'],
+            description: 'Select version of PXC operator',
+            name: 'PXC_OPERATOR_VERSION')
+        choice(
+            choices: ['none', 'v1.8.0', 'v1.9.0', 'v1.10.0', 'v1.11.0', 'v1.12.0'],
```

Comment:
Could you rename this none to latest? For better understanding what the job is doing?

Reply:
With 'none' the job doesn't install latest operators, it installs none so imho 'none' is accurate